docs/fpc-registration-new.puml

```diff
@@ -0,0 +1,87 @@
+@startuml
+
+'- force wrapping of longer text
+'skinparam wrapWidth 400
+
+!pragma teoz true
+
+
+title Fabric Private Chaincode - Enclave registry - New approach
+
+actor	Admin order 10
+participant ECC order 20
+participant ECC_Manager order 30
+participant ERCC order 60
+
+note right ERCC
+  ERCC can be **integrated** in ECC or as **seperate** chaincode
+  Integrated:
+    - ERCC and ECC share endorsement policy 
+    - attestations stored ECC namespace
+  Separate:
+    - flexible endorsement policies (ERCC and ECC can have different)
+    - attestations stored in ERCC namespace;
+end note
+
+entity IAS order 50
+collections Orderer order 70
+collections Peers order 80
```

Comment:
We really show it only for one peer right now.  At some level i think having at least two peers would help understanding all implications?

On a related note, as key-mgmt is a core part of ERCC, it would be nice to also add these flows?

Reply:
definitely, though it would be better to have it in a separate diagram

Reply:
sure, didn't necessarily mean the same diagram, just nice to have the "whole" picture of ercc as key-mgmt is a key-dist is a key part of it.  What could be also potentially useful is a simple class diagram of the ERCC (and, maybe ECM) state?